odbc/src/api.rs

```diff
@@ -408,39 +474,61 @@ pub extern "C" fn SQLDriversW(
     _drvr_attr_max: SmallInt,
     _out_drvr_attr: *mut SmallInt,
 ) -> SqlReturn {
-    unimplemented!()
+    unsafe {
+        (*(henv as *mut MongoHandle))
+            .add_diag_info(ODBCError::Unimplemented("SQLDriversW".to_string()));
+        SqlReturn::ERROR
+    }
 }
 
 #[no_mangle]
 pub extern "C" fn SQLEndTran(
     _handle_type: HandleType,
-    _handle: Handle,
+    handle: Handle,
     _completion_type: CompletionType,
 ) -> SqlReturn {
-    unimplemented!()
+    unsafe {
+        (*(handle as *mut MongoHandle))
```

Comment:
You should still return SQL_SUCCESS though 😅

Reply:
Ryan suggested removing the implementation for this function since it's not exactly unsupported! [Link](https://github.com/mongodb/mongo-odbc-driver/pull/3/files/10fb916131533dc8ab7d3333da5fac008d102cc0#r780487623) to comment